scripts/sdk-create-pr.sh

```diff
@@ -54,11 +58,15 @@ rm -rf ./*
 cp -a ${work_dir}/sdk_to_push/. ./
 
 # Create PR with new SDK if there are changes
-git switch -c "$1"
+if [[ "$REPO_BRANCH" != "main" ]]; then
+    git switch -c "$REPO_BRANCH"
+fi
 git add -A
-if git commit -m "$2"; then # Commit will fail if it doesn't contain any changes
-    git push origin "$1"
-    gh pr create --title "$3" --body "$4" --head "$1" --base "main"
+if git commit -m "$COMMIT_MESSAGE"; then # Commit will fail if it doesn't contain any changes
+    git push origin "$REPO_BRANCH"
```

Comment:
But if you are pushing to main, you'll need to add `--force` 
And give permissions to the technical user on ADO to bypass policies when pushing

Reply:
I'll see about the permissions, but why do we need to add `--force`? Isn't it a normal `push`?

Reply:
But push to main is not allowed, the branch is protected. So if you don't pass the `--force` flag it will throw error

Reply:
Ah I though the bypassing of policies would allow that already

Reply:
Tested and it worked